### ISSUE TITLE
The comparsion between per-pixel classification and masks classification.

### ISSUE THREAD

**Issue by ydhongHIT** [facebookresearch/MaskFormer#53]
I want to ask if the differences between fixed matching mask classification and per-pixel classification lie in sigmoid and loss function. Because I think the classification branch is useless on this occasion. That is to say, the improvement owing to replacing softmax and cross entropy Loss with sigmoid and your mask loss.

**Comment by bowenc0221** [comment]
![image](https://user-images.githubusercontent.com/25308473/144877303-4e537fdc-1304-4075-a324-3ba7b421f8bf.png)
